smithy-codegen-core/src/test/java/software/amazon/smithy/codegen/core/TraceFileTest.java

```diff
@@ -0,0 +1,60 @@
+package software.amazon.smithy.codegen.core;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import software.amazon.smithy.model.node.ExpectationNotMetException;
+import software.amazon.smithy.model.shapes.ShapeId;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+class TraceFileTest {
```

Comment:
Your two `TraceFile` test classes should probably be combined, since they're testing the functionality of the same class.